doc/tutorials/introduction/display_image/display_image.markdown

```diff
@@ -29,35 +42,39 @@ content of each individual module is declared.
 You'll almost always end up using the:
 
 -   *core* section, as here are defined the basic building blocks of the library
--   *highgui* module, as this contains the functions for input and output operations
-
-@snippet cpp/tutorial_code/introduction/display_image/display_image.cpp includes
-
-We also include the *iostream* to facilitate console line output and input. To avoid data structure
-and function name conflicts with other libraries, OpenCV has its own namespace: *cv*. To avoid the
-need appending prior each of these the *cv::* keyword you can import the namespace in the whole file
-by using the lines:
-
-@snippet cpp/tutorial_code/introduction/display_image/display_image.cpp namespace
+-   *imgcodecs* module, which provides functions for reading and writing
+-   *highgui* module, as this contains the functions to show an image in a window
 
-This is true for the STL library too (used for console I/O). Now, let's analyze the *main* function.
-We start up assuring that we acquire a valid image name argument from the command line. Otherwise
-take a picture by default: "HappyFish.jpg".
+We also include the *iostream* to facilitate console line output and input.
 
-@snippet cpp/tutorial_code/introduction/display_image/display_image.cpp load
+@snippet cpp/tutorial_code/introduction/display_image/display_image.cpp includes
+@end_toggle
 
-Then create a *Mat* object that will store the data of the loaded image.
+@add_toggle_python
+As a first step, the OpenCV python library is imported.
+The proper way to do this is to additionally assign it the name *cv*, which is used in the following to reference the library.
 
-@snippet cpp/tutorial_code/introduction/display_image/display_image.cpp mat
+@snippet samples/python/tutorial_code/introduction/display_image/display_image.py includes
+@end_toggle
 
-Now we call the @ref cv::imread function which loads the image name specified by the first argument
-(*argv[1]*). The second argument specifies the format in what we want the image. This may be:
+Now, let's analyze the main code.
+As a first step, we read the image "starry_night.jpg" from the OpenCV samples.
+In order to do so, a call to the @ref cv::imread function loads the image using the file path specified by the first argument.
+The second argument is optional and specifies the format in which we want the image. This may be:
 
 -   IMREAD_UNCHANGED (\<0) loads the image as is (including the alpha channel if present)
 -   IMREAD_GRAYSCALE ( 0) loads the image as an intensity one
--   IMREAD_COLOR (\>0) loads the image in the RGB format
+-   IMREAD_COLOR (\>0) loads the image in the RGB format. This is the default that is used here.
```

Comment:
Please remove `<0, 0, >0` notes - this is not correct anymore.

---

RGB => BGR 8-bit

---

`**default**` to make it bold?

---

Also I would suggest to reorder values: IMREAD_COLOR, IMREAD_GRAYSCALE, IMREAD_UNCHANGED